mod-invoice-storage/schemas/voucher.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "voucher",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID of this voucher",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "systemCurrency": {
+      "description": "The currency in which the voucher was paid",
+      "type": "string"
+    },
+    "invoiceCurrency": {
+      "description": "The currency in which invoice was originally defined",
+      "type": "string"
+    },
+    "exchangeRate": {
+      "description": "Exchange rate used to generate transaction",
+      "type": "number"
+    },
+    "exportToAccounting": {
+      "description": "Export this payment record to a financial system",
+      "type": "boolean"
+    },
+    "voucherType":{
+      "description": "Indicates type of payment to account",
+      "type": "string",
+      "enum": [
+        "Payment",
+        "Pre-Payment",
+        "Credit",
+        "Voucher"
+      ]
+    },
+    "voucherStatus":{
+      "description": "status of the voucher",
+      "type": "string",
+      "enum": [
+        "Awaiting Payment",
+        "Paid"
+      ]
+    },
+    "voucherAmount": {
+      "description": "The amount that is actually paid",
+      "type": "number"
+    },
+    "voucherDate": {
+      "description": "The date the invoice status changes to approved",
+      "type": "string",
+      "format": "date-time"
+    },
+    "voucherNumber": {
```

Comment:
As per interface field documentation, voucherNo can be [R36001, R36002]. Probably have a pattern to this field so it can be validated?

Reply:
As interface document is being worked on everyday, it was decided that new changes will be addressed later and this PR was created taking reference of 5/5 version. However, I think there is no fixed format for the number other than that it must be alpha-numeric

Reply:
The only pattern we can use here is to limit this to alpha-numeric... no special characters.

Reply:
Added pattern